images/pod-utilities/Dockerfile

```
@@ -2,32 +2,34 @@ ARG POD_UTILITY=clonerefs
 FROM ppc64le/ubuntu:18.04 as builder
 
 ARG UBUNTU_VERSION=18.04
-ARG BAZEL_VERSION=3.1.0
 ARG DEBIAN_FRONTEND=noninteractive
 ARG POD_UTILITY
 ARG CO_COMMIT=master
+ARG GOLANG_VERSION=1.15.7
+ENV PATH /usr/local/go/bin:$PATH
 
-COPY nodejs.patch /tmp
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:$PATH
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
 
 RUN apt-get update && apt-get install -y \
-    default-jdk \
     g++ \
     git \
     python \
     python3 \
     python3-dev \
     wget
 
-RUN wget --no-verbose https://oplab9.parqtec.unicamp.br/pub/ppc64el/bazel/ubuntu_${UBUNTU_VERSION}/bazel_bin_ppc64le_${BAZEL_VERSION} \
-    && mv bazel_bin_ppc64le_${BAZEL_VERSION} /usr/local/bin/bazel \
-    && chmod +x /usr/local/bin/bazel
+RUN wget -O go.tgz https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-ppc64le.tar.gz \
+    && tar -C /usr/local -xzf go.tgz \
+    && rm go.tgz \
+    && go version
 
 RUN git clone https://github.com/kubernetes/test-infra.git \
     && cd test-infra \
     && git checkout ${CO_COMMIT} \
-    && patch -p1 < /tmp/nodejs.patch \
-    && bazel build //prow/cmd/${POD_UTILITY} \
-    && cp bazel-bin/prow/cmd/${POD_UTILITY}/linux_ppc64le_pure_stripped/${POD_UTILITY} /usr/local/bin/${POD_UTILITY}
+    && go build ./prow/cmd/${POD_UTILITY} \
+    && cp ${POD_UTILITY} /usr/local/bin/${POD_UTILITY}
```

```suggestion
    && go build -o /usr/local/bin/${POD_UTILITY} ./prow/cmd/${POD_UTILITY}
```

you can directly use the -o option with `go build` command instead of an additional cp command

---

Done.